cachi2/core/package_managers/yarn_classic/main.py

```diff
@@ -91,11 +90,56 @@ def _generate_build_environment_variables() -> list[EnvironmentVariable]:
     return [EnvironmentVariable(name=key, value=value) for key, value in env_vars.items()]
 
 
+def _check_pnp_installs(project: Project) -> None:
+    if project.is_pnp_install:
+        raise PackageRejected(
+            reason=("Yarn PnP install detected; PnP installs are unsupported by cachi2"),
+            solution=(
+                "Please convert your project to a regular install-based one.\n"
+                "If you use Yarn's PnP, please remove `installConfig.pnp: true`"
+                " from 'package.json', any file(s) with glob name '*.pnp.cjs',"
+                " and any 'node_modules' directories."
+            ),
+        )
+
+
+def _verify_repository(project: Project) -> None:
+    _check_pnp_installs(project)
+    # _check_lockfile(project)
+
+    return None
+
+
+def _resolve_yarn_project(project: Project, output_dir: RootedPath) -> list[Component]:  # type: ignore
+    # Temporary type ignore
+    """Process a request for a single yarn source directory.
+
+    :param project: the directory to be processed.
+    :param output_dir: the directory where the prefetched dependencies will be placed.
+    :raises PackageManagerError: if fetching dependencies fails
+    """
+    log.info(f"Fetching the yarn-classic dependencies at the subpath {project.source_dir}")
+
+    _verify_repository(project)
+
+    # Placeholders for implementations in other PRs
+    # _set_yarnrc_configuration(project, output_dir)
+    # packages = resolve_packages(project.source_dir)
+    # _fetch_dependencies(project.source_dir)
+
+    # return create_components(packages, project, output_dir)
+
+
 def _verify_corepack_yarn_version(source_dir: RootedPath, env: dict[str, str]) -> None:
     """Verify that corepack installed the correct version of yarn by checking `yarn --version`."""
     installed_yarn_version = extract_yarn_version_from_env(source_dir, env)
 
-    if installed_yarn_version not in VersionsRange("1.22.0", "2.0.0"):
```

Comment:
This looks like a conflict resolution gone wrong, [a00a8320](https://github.com/containerbuildsystem/cachi2/commit/a00a8320625a093700522f9f951a5d93801de39b) introduced a reverse change (L8 in the same module too), could you please check?

Reply:
Rats, I must've resolved in the wrong direction...

Reply:
Fixed.